src/Kernel/Magic/LsOpenMagic.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp.Jupyter;
+
+namespace Microsoft.Quantum.IQSharp.Kernel
+{
+    /// <summary>
+    ///     A magic command that lists any open namespaces and their aliases.
+    /// </summary>
+    public class LsOpenMagic : AbstractMagic
+    {
+        /// <summary>
+        ///     Constructs an instance of %lsopen given an instance of the
+        ///     compiler service.
+        /// </summary>
+        public LsOpenMagic(ICompilerService compiler) : base(
+            "lsopen",
+            new Documentation
+            {
+                Summary = "Lists currently opened namespaces and their aliases.",
+                Description = @"
+                    This magic command lists any namespaces have been made
+                    available using `open` statements, along with any aliases
+                    that may have been assigned to those namespaces.
+                ".Dedent(),
+                Examples = new []
+                {
+                    @"
+                        Print a list of all currently opened namespaces:
+                        ```
+                        In []: %lsopen
+                        Out[]: Namespace                         Alias
+                               --------------------------------- -----------
+                               dump.basisStateLabelingConvention ""BigEndian""
+                               dump.truncateSmallAmplitudes      true
```

Comment:
```suggestion
                               Microsoft.Quantum.Canon
                               Microsoft.Quantum.Intrinsic
```

Reply:
Thanks for catching this! I fixed it locally and pushed, resolving accordingly.